### PR TITLE
[pt-br] Categorias de conteúdo (Content Categories) links adjusted

### LINKED ISSUE #17531: [pt-br] Categorias de conteúdo (Content Categories) links broken
The link to the *Content Categories* page is [*incorrect*](https://developer.mozilla.org/pt-BR/docs/HTML/Categorias_de_conteúdo) in [*some pages*](https://github.com/search?q=repo%3Amdn%2Ftranslated-content%20Categorias_de_conte%C3%BAdo&type=code), it should be [*/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo*](https://developer.mozilla.org/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo).

This happen in:
- [`<option>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/option#sum%C3%A1rio)
- [`<input>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/input)
- [`<select>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/select#contexto_de_uso)

### PR DESCRIPTION
**Pull request by diegoborbadev**
### Description

As mentioned in issue #17531, some links referencing the "Categorias de conteúdo (Content Categories)" section were incorrect and have been corrected.

### Motivation

Direct readers to the correct address.

### Related issues and pull requests

Fixes #17531

### CODE REVIEW

File: files/pt-br/web/html/element/input/index.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ slug: Web/HTML/Element/input
 
 O **elemento HTML `<input>`** é usado para criar controles interativos para formulários baseados na web para receber dados do usuário. A semântica de um `<input>` varia consideravelmente dependendo do valor de seu atributo `type`.
 
-- _[Categorias de conteúdo](/pt-BR/docs/HTML/Categorias_de_conteúdo)_ [Conteúdo de fluxo](/pt-BR/docs/HTML/Categorias_de_conteúdo#Conteúdo_de_fluxo), listado, enviável, resetável, elemento associado a formulário, [conteúdo fraseado](/pt-BR/docs/HTML/Categorias_de_conteúdo#Conteúdo_fraseado).
+- _[Categorias de conteúdo](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo)_ [Conteúdo de fluxo](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_de_fluxo), listado, enviável, resetável, elemento associado a formulário, [conteúdo fraseado](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_fraseado).
```
Comment (yin1999) [inline files/pt-br/web/html/element/input/index.md:8]:
```suggestion
- _[Categorias de conteúdo](/pt-BR/docs/Web/HTML/Content_categories)_ [Conteúdo de fluxo](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_de_fluxo), listado, enviável, resetável, elemento associado a formulário, [conteúdo fraseado](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_fraseado).
```

File: files/pt-br/web/html/element/input/index.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ slug: Web/HTML/Element/input
...
   Se o atributo [`type`](/pt-BR/docs/Web/HTML/Element/input#type) não tiver o valor `hidden`, elemento rotulável, conteúdo palpável.
 - _Conteúdo permitido_ Nenhum, este é um {{Glossary("elemento vazio")}}.
 - _Omissão de tags_ Deve ter uma tag de abertura e não deve ter uma tag de fechamento.
-- _Elementos pais permitidos_ Qualquer elemento que aceite [conteúdo fraseado](/pt-BR/docs/HTML/Categorias_de_conteúdo#Conteúdo_fraseado).
+- _Elementos pais permitidos_ Qualquer elemento que aceite [conteúdo fraseado](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_fraseado).
```
Comment (yin1999) [inline files/pt-br/web/html/element/input/index.md:12]:
```suggestion
- _Elementos pais permitidos_ Qualquer elemento que aceite [conteúdo fraseado](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_fraseado).
```

File: files/pt-br/web/html/element/option/index.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ slug: Web/HTML/Element/option
 
 Em um formulário Web, o **elemento HTML** `<option>` é usado para criar um controle que representa um item dentro de um elemento HTML5 {{HTMLElement("select")}}, {{HTMLElement("optgroup")}} ou {{HTMLElement("datalist")}}.
 
-- _[Categorias de conteúdo](/pt-BR/docs/Web/HTML/Categorias_de_conteúdo)_ Nenhuma.
+- _[Categorias de conteúdo](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo)_ Nenhuma.
```
Comment (yin1999) [inline files/pt-br/web/html/element/option/index.md:10]:
```suggestion
- _[Categorias de conteúdo](/pt-BR/docs/Web/HTML/Content_categories)_ Nenhuma.
```

File: files/pt-br/web/html/element/select/index.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ O elemento HTML _select_ (\<select>) representa um controle que apresenta um men
...
+| [Categorias de conteúdo](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo) | [conteúdo fluido](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_fluido), [conteúdo fraseado](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_fraseado), [conteúdo interativo](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Conteúdo_interativo), [listado](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Listado_em_formulários), [rotulável](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Rotulável_em_formulários), [resetável](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Resetável_em_formulários), e [enviável](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Enviável_em_formulários) [associado a formulários](/pt-BR/docs/Web/Guide/HTML/Categorias_de_conteudo#Associado_a_formulários) |
```
Comment (yin1999) [inline files/pt-br/web/html/element/select/index.md:12]:
```suggestion
| [Categorias de conteúdo](/pt-BR/docs/Web/HTML/Content_categories) | [conteúdo fluido](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_de_fluxo), [conteúdo fraseado](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_fraseado), [conteúdo interativo](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_interativo), [listado](/pt-BR/docs/Web/HTML/Content_categories#listed), [rotulável](/pt-BR/docs/Web/HTML/Content_categories#labelable), [resetável](/pt-BR/docs/Web/HTML/Content_categories#resettable), e [enviável](/pt-BR/docs/Web/HTML/Content_categories#submittable) [associado a formulários](/pt-BR/docs/Web/HTML/Content_categories#conteúdo_associado_ao_form) |
```

**Review by yin1999 — COMMENTED** [review]
I believe we can fix those links further

**Comment by diegoborbadev** [comment]
> I believe we can fix those links further

Thank you @yin1999! Now it's even better.

**Review by josielrocha — APPROVED** [review]

**Comment by josielrocha** [comment]
Obrigado pela ajuda @diegoborbadev

**Pull request merged**